idaes/models_extra/power_generation/costing/power_plant_costing.py

```diff
@@ -74,14 +74,150 @@
     between 90% and 97.7% capture rate. It is only valid for a flue gas CO2
     concentration of 4%. The additional cost component of the CO2 system is
     in account 5.1.b and remains unchanged.
- """
+
+    - 6.1 can be used to cost the MEA solvent-based carbon capture system, 
+    using the reference: Kangkang Li, Ashleigh Cousins, Hai You, Paul Feron,
+    Weilang Luo, Jian Chen (2016). Systematic study of aqueous monoethanolamine
+    (MEA)-based CO2 capture process: Techno-economic assessment of the MEA
+    process and its improvements. Applied Energy, 165, 648-659.
```

Comment:
It would be good to mention if there is a valid range of capture percentages or flue gas CO2 concentrations for these costs.